packages/cms/lib/modules/resource-form-widgets/public/js/map.js

```diff
@@ -2,7 +2,7 @@ apos.define('resource-form-widgets', {
     extend: 'map-widgets',
     construct: function(self, options) {
         self.playAfterlibsLoaded = function($widget, data, options) {
-            var mapConfig = typeof resourceMapConfig !== 'undefined' && resourceMapConfig ? resourceMapConfig : {};
+            var mapConfig = typeof resourceMapConfig !== 'undefined' && resourceMapConfig ? resourceMapConfig : data.mapConfig;
```

Comment:
Wellicht nog een check of data.mapConfig wel bestaat? Anders terugvallen op `{}`

Reply:
Op de regel daarna wordt al gecheckt of mapConfig bestaat. Dus als `data.mapConfig` undefined is dan zal de map niet geladen worden. `data` zal altijd een object zijn die vanuit apostrophe komt. 
```
if (mapConfig) {
    self.createMap(mapConfig);
    self.addPolygon(mapConfig);
    self.setIdeaMarker(mapConfig);
    self.addFormEventListeners(mapConfig);
    self.center();
}
```